src/utils/storage.ts

```diff
@@ -0,0 +1,25 @@
+export const getStorage = (key: string, defaultValue = '') => {
+  try {
+    const value = localStorage.getItem(key);
+    return value ? JSON.parse(value) : defaultValue;
+  } catch (error) {
+    console.error(error);
+    return defaultValue;
+  }
+};
+
+export const setStorage = (key: string, value: string) => {
+  try {
+    localStorage.setItem(key, JSON.stringify(value));
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+export const removeStorage = (key: string) => {
```

Comment:
```suggestion
export const removeLocalStorage = (key: string) => {
```
`localStorage`임을 명시해주는 것이 좋을 것 같습니다!